### ISSUE TITLE
Site should pass a Wave Audit for WCAG 2.0

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/special_collections#213]
Right now there are quite a few violations http://wave.webaim.org/report#/rbscbeta.princeton.edu. This might not be a release blocker but it should resolved before we get to the "phase" two period for the rolled we discussed with Gabriel.
